src/Console/Command.php

```diff
@@ -104,6 +105,29 @@ public function getName(): string
         return $this->name;
     }
 
+    /**
+     * Get the command name.
+     *
+     * Returns the command name based on class name.
+     * For app / plugin command with class name `UpdateTableCommand` the command
+     * name returned would be `'update_table'`.
+     * If same command class is under `Cake` namespace then it will return `'update table'`.
+     *
+     * @return string
+     */
+    public static function defaultName(): string
+    {
+        $pos = strrpos(static::class, '\\');
+        $name = substr(static::class, $pos + 1, -7);
+        $name = Inflector::underscore($name);
+
+        if (substr(static::class, 0, 4) === 'Cake') {
```

Comment:
I think it would be best to remove this core specific code and simply add `defaultName()` to all core commands which return static strings. That would be a good micro optimization too.

Reply:
yes we can do it,  just there are 22 command classes :)
I'll add them

Reply:
Actually adding static name is necessary only for multi worded core commands. I have done that :slightly_smiling_face:.